src/components/Search.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import '../styles/Search.css';
+
+export default ({ handleInputChange, inputText, handleClearInput }) => {
+  return (
+    <>
+      <input
+        className="search-input"
+        onChange={handleInputChange}
+        value={inputText}
+        type="text"
+        placeholder="search your item..."
+        //NOTE: Try to make clear button more accessible
+      />
+      {inputText.length ? (
```

Comment:
More of a stylistic preference, but if you only want want a component to render on one condition (since here the ternary's second evaluation is producing null), you can use what is called short circuit evaluation using `&&`

#5 in this article: https://scotch.io/tutorials/7-ways-to-implement-conditional-rendering-in-react-applications#toc-5-logical-short-circuit-evaluation-with-

Reply:
Yes, great suggestion. And just learned a very interesting fact: 
I tried to use short circuit evaluation but since the condition is evaluated based on `inputText.length`, there is 0 (zero) rendered below the search bar. The reason is that the first condition is evaluated as falsy, which returns 0 however, 0(zero) is not ignored (unlike `null`, `undefined` and `false`) and is still rendered on the DOM. 
The solution here was to evaluate condition with > (more) than 0. `{inputText.length >0  && <button className="clear-button" onClick={handleClearInput}>X</button>}`. 

Here is a [StackOverflow](https://stackoverflow.com/questions/60652656/why-0-is-rendered-on-short-circuit-evaluation-of-array-length) discussion. I thought it would be interesting to share.